### DISCUSSION TITLE
am trying to understand the different networks

### DISCUSSION THREAD

**Discussion by omegajudith** [Cyfrin/foundry-full-course-cu#2580] [General]
what are networks? I see Sepolia. sorry, I am new and am in  the first section of the course

**Comment by EngrPips** [comment]
Keep going, Patrick explained everything in the course.

**Comment by Napcalx** [comment]
They are basically the different Blockchains or layer 2 you'd be deploying to.

**Comment by EngrPips** [comment]
Interesting
